docs/antrea-network-policy.md

```diff
@@ -1037,13 +887,82 @@ spec:
         name: AllowToCoreDNS
 ```
 
-The above example allows all Pods from Namespace "default" to connect to all "core-dns"
+The above example allows all Pods from Namespace "default" to connect to all "kube-dns"
 Pods from Namespace "kube-system" on TCP port 53.
 
-## FQDN based filtering
+### Matching self Namespace for appliedTos
```

Comment:
section header is hard to understand IMO

maybe "Selecting Pods in the Namespace in which a policy rule is applied with `Self`"

Reply:
I would like to keep it a little more concise. How does "Selecting Pods in the same Namespace with `Self`" sound?

Reply:
Not related to this PR, but if we extend to `NotSelf`, shall we open a new section "### Selecting ... with `NotSelf`", or attach with this section since it introduces namespaces matching?

Reply:
We could rename the section to `Advanced Namespace matching` and list two sub-sections for `Self` and `NotSelf`?